src/utils/IdealGraphVisualizer/Util/src/main/java/com/sun/hotspot/igv/util/PropertiesConverter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package com.sun.hotspot.igv.util;
+
+import com.sun.hotspot.igv.data.Properties;
+import com.sun.hotspot.igv.data.Property;
+
+/**
+ *
+ * @author ksakata
+ */
+public class PropertiesConverter {
+    public static String convertToHTML(final Properties properties) {
+        StringBuilder sb = new StringBuilder("<html><body><table>");
+        for (Property p : properties) {
+            sb.append("<tr><td>")
+              .append(p.getName())
+              .append("</td><td>")
+              .append(p.getValue())
+              .append("</td></tr>");
+        }
```

Comment:
```suggestion
        StringBuilder sb = new StringBuilder("<html><body><table cellpadding="0" cellspacing="0">");
        for (Property p : properties) {
            sb.append("<tr><td>")
              .append(p.getName())
              .append("</td><td width="10"></td><td>")
              .append(p.getValue())
              .append("</td></tr>");
        }
```

Perhaps the spacing the spacing between the text could be reduced? 
*Currently:*
![current spacing](https://user-images.githubusercontent.com/71546117/183066426-6b057e12-4d91-4b8c-ad97-8c8923d91ff8.png)


*Suggestion:*
![spacing suggestion](https://user-images.githubusercontent.com/71546117/183066296-25b42459-2ba2-40f5-b020-bdc7b38e9e81.png)